prog/vm/nexus.rb

```diff
@@ -11,7 +11,7 @@ class Prog::Vm::Nexus < Prog::Base
 
   def self.assemble(public_key, project_id, name: nil, size: "m5a.2x",
     unix_user: "ubi", location: "hetzner-hel1", boot_image: "ubuntu-jammy",
-    private_subnet_id: nil, nic_id: nil, storage_size_gib: 20, storage_encrypted: false)
+    private_subnet_ubid: nil, nic_ubid: nil, storage_size_gib: 20, storage_encrypted: false)
```

Comment:
Should we get ubid as params or convert ubid to uuid at route file?

Reply:
This is a good question. I was thinking we would start using ubid everywhere but now that you say this. In Route, converting it and if it's not valid, simply failing there is a better idea. I will change it accordingly. 👍